packages/social-trader/src/components/notifications/components/notification/notification.tsx

```diff
@@ -38,32 +28,46 @@ import {
   $paddingXxsmall
 } from "utils/style/sizes";
 
-enum TYPE {
-  PROFILE = "profile",
-  PLATFORM = "platform"
+import { getColor, getLink, getStaticIconUrl } from "./notification-helpers";
+
+export enum TYPE {
+  PROFILE = "Profile",
+  PLATFORM = "Platform",
+  SOCIAL = "Social",
+  ASSET = "Asset",
+  TRADING_ACCOUNT = "TradingAccount",
+  USER = "User",
+  SIGNAL = "Signal",
+  PLATFORM_ASSET = "PlatformAsset"
+}
+
+export enum LINKS {
```

Comment:
What happens if the backend changes values of the `location` field in NotificationLocationViewModel?
Maybe this enum should be declared in the API?

Reply:
Backend said me there is no chance to use enum, because mobile app crashes.